### PR TITLE
[auditv2] Fix more e2e tests

### PR DESCRIPTION
**Pull request by iethree**
### Description

> **Note**
> The backend changes in this PR have already been reviewed, they related to Auditv2 tables showing up in Auditv1

e2e tests fixed:
- most were things that just didn't resolve conflicts with master correctly
- a couple were things that we can only test on OSS now due to the existence of the IA db, collections, questions, and models
- a couple were just stupid fragile tests that failed because column names got longer 😢 

This should finally get `auditv2-main` back to green ✅

### CODE REVIEW

File: e2e/test/scenarios/models/reproductions/19776-data-picker-not-displayed-after-archiving-model.cy.spec.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { restore, popover } from "e2e/support/helpers";
+import { restore, popover, describeOSS } from "e2e/support/helpers";
 
 const modelName = "Orders Model";
 
-describe("issue 19776", () => {
+// this is only testable in OSS because EE always has models from auditv2
+describeOSS("issue 19776", { tags: "@OSS" }, () => {
```
Comment (iethree) [inline e2e/test/scenarios/models/reproductions/19776-data-picker-not-displayed-after-archiving-model.cy.spec.js:6]: @nemanjaglumac , a couple questions

- is `describeOSS` deprecated? if so, we should just remove it, right?
- does adding the `@OSS` tag here to make sure it only runs on a version of metabase without audit models make sense to you? Anything I'm missing here?

Comment (nemanjaglumac) [reply e2e/test/scenarios/models/reproductions/19776-data-picker-not-displayed-after-archiving-model.cy.spec.js:6]: - `describeOSS` will skip tests if Metabase version is not OSS.
- Tag `@OSS` doesn't guarantee anything unless you remember to use it in a Cypress run command.

Does this make sense?
And does it answer your question?

File: e2e/test/scenarios/visualizations/table-column-settings.cy.spec.js [hunk 2]
```diff
@@ -35,6 +35,10 @@ const tableQuestionWithJoinOnQuestion = card => ({
   display: "table",
   query: {
     "source-table": ORDERS_ID,
+    fields: [
+      ["field", ORDERS.ID, null],
+      ["field", ORDERS.TAX, null],
+    ],
```
Comment (nemanjaglumac) [inline e2e/test/scenarios/visualizations/table-column-settings.cy.spec.js:41]: Such a beauty ❤️

File: e2e/test/scenarios/joins/reproductions/15578-remapped-values-joins.cy.spec.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ describe("issue 15578", () => {
 
     queryBuilderMain().within(() => {
       cy.findByText("Category").should("be.visible");
-      cy.findByText("Question 5 → ID").should("be.visible");
+      cy.findByText(/Question \d+ → ID/).should("be.visible");
```
Comment (nemanjaglumac) [inline e2e/test/scenarios/joins/reproductions/15578-remapped-values-joins.cy.spec.js:59]: 👌

**Review by nemanjaglumac — APPROVED** [review]
Awesome!
Thank you for 🧹

File: e2e/test/scenarios/visualizations/table-column-settings.cy.spec.js (hunk 2 above)
Comment (iethree) [reply e2e/test/scenarios/visualizations/table-column-settings.cy.spec.js:41]: thanks for the idea!

**Pull request merged**